docs/data/routes/docs/fundamentals/cli/en.md

```diff
@@ -5,28 +5,85 @@ title: JSS CLI
 ---
 # JSS CLI
 
-During JSS development, you will be commonly using scripts provided by the _JSS CLI_ (command line interface). These scripts can be run from within a JSS application at your command line of choice (bash, PowerShell, cmd, etc).
+## Env File Support
 
-## Frequently used scripts
+The CLI supports loading environment variables from `.env` files. Use a `.env` file to populate environment variables such as `NODE_EXTRA_CA_CERTS` and `NODE_TLS_REJECT_UNAUTHORIZED`, which you may have already configured in a `.env` file for communication with your Sitecore instance.
 
-These are the main scripts you will be using frequently during JSS development.
-These were partially covered in [Quick start](/docs/client-frameworks/getting-started/quick-start).
+Variables will be loaded from `.env` files with the following priority:
 
-### `jss create`
-Creates a JSS application. You must provide an application name and a framework. 
+* `.env.${NODE_ENV}.local`
+* `.env.local` (unless `NODE_ENV` is "test")
+* `.env.${NODE_ENV}`
+* `.env`
+
+The CLI will automatically expand variables (`$VAR`) inside your `.env*` files. This allows you to reference other values. If you are attempting to use a `$` in a variable value, it needs to be escaped with a backslash (`\`).
 
 ```
+PASSWORD=b
+
+# becomes 'adminb'
+EXPANDED=admin$PASSWORD
+
+# becomes 'admin$PASSWORD'
+ESCAPED=admin\$PASSWORD
+```
+
+## JSS commands
+
+During JSS development, you will commonly use scripts provided by the _JSS CLI_ (command-line interface). All scripts you need to run during development are defined either via the `jss` CLI command (use `--help` for options) or package scripts in `package.json`.
```

Comment:
We have commands defined in the sitecore-jss-cli base package (create, setup, deploy), and then there are client-controlled commands like `build` and `scaffold`, which can look like jss scripts because `jss` in the command line acts as a wrapper for `npm run`, but they are totally up to the client.

In this section, these 2 types of scripts are being discussed together, and I'm worried that this may be misleading. I think we should document commands from the base package separately, and then have a section for commands defined in the examples. The tricky part is that *some* commands defined in the samples differ by framework... it might be best to save this update for the new docs? Then each framework could have it's own CLI reference and we could utilize that doc snippet functionality to repeat shared docs.

Reply:
I think you are right about saving the update for the new docs. We won't have time to add and verify all framework-specific differences on env, and client commands.